samples/04_gis_analysts_data_scientists/temperature_forecast_using_time_series_data.ipynb

```diff
@@ -69,7 +69,7 @@
   },
```

Comment:
<p>Please remove the <code>england_map.zoom = 6</code>from these lines and place it in a cell below the map by itself.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>